src/User/user.utils.js

```diff
@@ -0,0 +1,22 @@
+import jsonwebtoken from "jsonwebtoken";
+import jwt from "jsonwebtoken";
+import client from "../client";
+
+export const getUser = async (Authorization) => {
```

Comment:
変数名は先頭大文字はあまりしなさそうです！

```suggestion
export const getUser = async (authorization) => {
```